nodejs/week1/meals/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+/node_modules
```

Comment:
This should be node_modules instead of /node_modules.

When you use /node_modules it refers to the node_modules folder in the same folder as the .gitignore, and none other. This can cause issues if you have node_modules in any subfolder.